src/util-memcmp.c

```diff
@@ -38,9 +38,7 @@ static int MemcmpTest01 (void)
     uint8_t a[] = "abcd";
     uint8_t b[] = "abcd";
 
-    if (SCMemcmp(a, b, sizeof(a)-1) != 0)
-        return 0;
-
+    FAIL_IF(SCMemcmp(a, b, sizeof(a) - 1) != 0);
     return 1;
```

Comment:
All `return 1` statements in unit tests should be replaced with `PASS` macro calls ;)